src/applications/vaos/tests/new-appointment/redux/helpers/formSubmitTransformers.unit.spec.js

```diff
@@ -769,4 +769,123 @@ describe('VAOS data transformation', () => {
       providerOption: '',
     });
   });
+
+  it('should transform form using provider selection into CC request when residential address is missing', () => {
+    const state = {
```

Comment:
This looks good. Just need to add the CCIteration toggle flag to the state
```suggestion
    const state = {
      featureToggles: {
        vaOnlineSchedulingCCIterations: true,
      },
```